packages/paste-website/src/pages/components/toast/index.mdx

```diff
@@ -163,6 +193,12 @@ Add an automatic time-out if all these conditions are true:
 
 We recommend giving users about 0.5 seconds per word before timing out a toast.
 
+#### Basic
+
+For a basic method of hiding and showing, you can just render a `<Toast />` inside of a `<ToastContainer />`.
+
+When showing multiple toasts, place the most recent toast at the bottom of the stack.
```

Comment:
I know what you mean but it reads weird haha.  At first I thought "at the top of the stack" and now I'm not sure sure. FIFO? New toasts go into the front of an array?